worldedit-core/build.gradle.kts

```diff
@@ -84,3 +87,28 @@ sourceSets {
         }
     }
 }
+
+val crowdinApiKey = "crowdin_apikey"
+
+if (project.hasProperty(crowdinApiKey)) {
+    tasks.named<UploadSourceFileTask>("crowdinUpload") {
+        apiKey = "${project.property(crowdinApiKey)}"
+        projectId = "worldedit-core"
+        files = arrayOf(
+                object {
```

Comment:
I feel like this would be better represented as a Map,
```gradle.kts
mapOf("name" to "string.json", "source" to "...")
```

Additionally, `source` would be shorter as `"${file("src/main/resources/lang/strings.json")}"`